src/Cascade.php

```diff
@@ -31,6 +31,33 @@ class Cascade
      * @var Config
      */
     protected static $config = null;
+    
+    /**
+     * Inherit on undefined logger option
+     *
+     * @var bool
+     */
+    protected static $inherit_on_undefined_logger = false;
```

Comment:
The convention for this is lowerCamelCase. Though, stick with what the file uses if it violates this.